File: android/src/main/kotlin/com/syncrhrohealth/bandpass_ble_connection/core/command/ScheduleGetCmd.kt
package com.syncrhrohealth.bandpass_ble_connection.core.command

import com.syncrhrohealth.bandpass_ble_connection.core.command.builder.CommandBuilder
import com.syncrhrohealth.bandpass_ble_connection.core.enum.CommandCode
import com.syncrhrohealth.bandpass_ble_connection.core.handler.DeviceHandler


object ScheduleGetCmd {
    fun send(handler: DeviceHandler) {
        val builder = CommandBuilder.build(CommandCode.PACKET_CMD_SCHEDULE_GET)
        handler.write(builder)
    }
}

File: ios/Classes/Core/Command/ScheduleGetCmd.swift
//
//  ScheduleGetCmd.swift
//  Runner
//
//  Created by MAC on 11/3/25.
//

import Foundation

struct ScheduleGetCmd {
    static func send(handler: DeviceHandler) {
        let builder = CommandBuilder.build(commandCode: .packetCmdScheduleGet)
        handler.write(data: builder)
    }
}
